projects/storefrontstyles/scss/components/checkout/steps/review/_review-submit.scss

```diff
@@ -34,14 +35,45 @@
       }
     }
 
+    .cx-review-payment-col {
+      border-inline-end: 1px solid var(--cx-color-text);
+    }
+
     .cx-review-summary-card {
+      display: flex;
+      justify-content: space-between;
+
       @include media-breakpoint-down(md) {
         background-color: theme-color('inverse');
         border-style: solid;
         border-width: 1px;
         border-color: var(--cx-color-light);
         min-height: auto;
       }
+
+      &:not(:last-of-type) {
+        border-bottom: 1px solid rgba(0, 0, 0, 0.5);
```

Comment:
should only be applied to mobile for `@include media-breakpoint-down(md)` as it makes it thicker with the already existing border for mobile

Reply:
I like the border bottom in desktop, but maybe need to check up with Bill with a screenshot since I don't see that in the mocks

Reply:
Oh yes, this is a secret mock that Bill sent me. I will update the page.

Reply:
And yes I showed it to Bill and got approval 😄